pkg/tsdb/loki/frame_test.go

```diff
@@ -0,0 +1,88 @@
+package loki
+
+import (
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFormatName(t *testing.T) {
+	t.Run("converting metric name", func(t *testing.T) {
+		metric := map[string]string{
+			"app":    "backend",
+			"device": "mobile",
+		}
+
+		query := &lokiQuery{
+			LegendFormat: "legend {{app}} {{ device }} {{broken}}",
+		}
+
+		require.Equal(t, "legend backend mobile ", formatName(metric, query))
+	})
+
+	t.Run("build full series name", func(t *testing.T) {
+		metric := map[string]string{
+			"__name__": "http_request_total",
```

Comment:
Loki doesn't have metrics so I think this test might be confusing because it gives an impression that we can have metrics there.

Reply:
yeah, i agree, done.